app/assets/src/stylesheets/02-molecules/forms.scss

```diff
@@ -7,6 +36,23 @@ input[type=checkbox] + label {
   margin-bottom: 0;
 }
 
-.field, form > .actions {
+.field form > .actions {
   margin: 1.5em auto;
-}
\ No newline at end of file
+}
+
+.actions {
+  display: grid;
+  width: 100%;
+  grid-template-columns: [secondary] 1fr [primary] 10rem;
+  grid-auto-rows: minmax(0, auto);
+}
+
+.actions--primary {
```

Comment:
I'm pretty sure you want flexbox for this rather than grid and that's why the markup and CSS are fighting you. Flexbox is for one-dimensional arrangements, grid is for two-dimensional ones.

Also, this is properly something more like `.form__actions__primary`, `.form__actions > .--primary`, or `.form__actions > $(all-buttons).--primary` -- when you visualize it as that, and flatten out the markup so it's:

```html
<div class='actions'>
  <button class='--continue'>Go on</button>
  <button class='--cancel'>Cancel</button>
</div>
```

then the CSS can probably become something closer to

```css
.form__actions {
  display: flex;
}

.form__actions > .--continue {
  justify-self: flex-end;
}

.form__actions > .--cancel {
  justify-self: flex-start;
}
```

I'm using `--continue` and `--cancel` there not because I necessarily think you should use those specifically, but more because the way you framed this in your issue description was about signaling "go vs back" to the user, and so I think we can and should get more precise with our naming than "primary/secondary" -- those are great when you don't know what kind of distinction you're trying to draw, and certainly 100% better than "left/right" like people so often fall back on, but in this case we know exactly what we're trying to do and so we have an opportunity to be precise in our naming.

Reply:
Nice! I can make that work. Grid was fighting me and I didn't know how/why